src/components/app/list-new/AppList.tsx

```diff
@@ -871,15 +871,17 @@ export default function AppList() {
                 rootClassName="app-create-model-wrapper"
                 onClick={() => setShowCreateNewAppSelectionModal(!showCreateNewAppSelectionModal)}
             >
-                <div className="app-create-child c-pointer" onClick={openDevtronAppCreateModel}>
-                    <AddIcon className="icon-dim-20 fcn-9" />
-                    <div className="ml-8">
-                        <strong>Custom app</strong>
-                        <div>
-                            Connect a git repository to deploy <br /> a custom application
+                {serverMode == SERVER_MODE.FULL && appCheckListRes?.result?.appChecklist && Object.values(appCheckListRes.result.appChecklist).every((check) => check)  && (
```

Comment:
on line no 43 we are initializing appCheckListRes so no need to '?' after appCheckListRes

Reply:
done